docs/reference/edot-collector/components/apmconfigextension.md

```diff
@@ -0,0 +1,115 @@
+---
+navigation_title: APM Config extension
+description: The APM Config extension is an OpenTelemetry Collector component that enables central configuration delivery for EDOT SDKs using the Open Agent Management Protocol (OpAMP).
+applies_to:
+  stack:
+  serverless:
+  observability:
+  product:
+    edot_collector:
+products:
+  - id: elastic-agent
+  - id: observability
+  - id: edot-collector
+---
+
+# APM Config extension
+
+The {{product.apm}} Config extension (`apmconfigextension`) enables central configuration for {{edot}} SDKs through the Open Agent Management Protocol (OpAMP). It connects the {{edot}} Collector to the Elastic configuration server ({{product.apm-server}} or {{fleet}} Server) so that configuration updates can be retrieved and applied dynamically.
+
+{applies_to}`stack: ga 9.1` This component is a core part of the {{edot}} Collector distribution and is required for enabling the [Central Configuration feature](docs-content://solutions/observability/apm/opentelemetry/edot-sdks-central-configuration.md) in {{product.observability}}.
```

Comment:
Not a core component at the moment: https://github.com/elastic/elastic-agent/blob/main/internal/pkg/otel/components.yml#L60